cost/azure/rightsize_compute_instances/README.md

```diff
@@ -64,6 +64,9 @@ For administrators [creating and managing credentials](https://docs.flexera.com/
 - [**Azure Resource Manager Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_109256743_1124668) (*provider=azure_rm*) which has the following permissions:
   - `Microsoft.Compute/virtualMachines/read`
   - `Microsoft.Compute/virtualMachines/write`*
+  - `Microsoft.Compute/virtualMachines/powerOff/action` *
+  - `Microsoft.Compute/virtualMachines/start/action` *
```

Comment:
There should be no space between ` and *

Reply:
Resolved